TeachingRecordSystem/tests/TeachingRecordSystem.TestCommon/TestData.CreatePersonProfessionalStatusBuilder.cs

```diff
@@ -0,0 +1,195 @@
+namespace TeachingRecordSystem.TestCommon;
+
+using TeachingRecordSystem.Core.DataStore.Postgres;
+using TeachingRecordSystem.Core.DataStore.Postgres.Models;
+
+public partial class TestData
+{
+    public class CreatePersonProfessionalStatusBuilder
+    {
+        // for reference data, create some nonsense values temporarily for the reference tables that aren't populated
```

Comment:
Is there a better way of adding some data while we wait for the real lookups?
This doesn't work when running multiple tests - I thought about using AutoFixture to generate a unique 4-letter code each time but there's maybe somewhere better to do this setup anyway